frontend/src/lib/components/TestJobLoader.svelte

```diff
@@ -13,6 +13,15 @@
 	export let jobUpdateLastFetch: Date | undefined = undefined
 	export let toastError = false
 	export let lazyLogs = false
+	// Will be set to number if job is not a flow
+	// If you want to find out progress of subjobs of a flow, check job.flow_status.progress
+	export let scriptProgress: number | undefined = undefined;
+
+	/// Last time asked for job progress
+	let lastTimeCheckedProgress: number | undefined = undefined; 
+
+	/// Delay in ms before frontend is pulling job's progress
+	const getProgressDelay: number = 3000;
```

Comment:
There should be 2 delays:
- One until we find our first progress (every 5s)
- Once we found our first progress, we can do it every second

Reply:
Ok, got it :+1:

Reply:
Changed: https://github.com/windmill-labs/windmill/pull/4373/commits/099d6d2ee354c4f1e43d93000b23bc57173b4af2